ontologies/vocabs.ttl

```diff
@@ -1158,8 +1142,10 @@ Tag for the government-national industry sector"""^^xsd:string ;
 
 gist:_IndustrySector_government-Public-Services
 	a gist:IndustrySector ;
-	rdfs:label "Government Public Services"^^xsd:string ;
-	skos:altLabel "Government-Public-Services"^^xsd:string ;
+	skos:altLabel
+		"Government Public Services"^^xsd:string ,
+		"Government-Public-Services"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?